lua/nvim-treesitter/parsers.lua

```diff
@@ -1211,6 +1211,15 @@ list.just = {
   maintainers = { "@Hubro" },
 }
 
+list.kcl = {
+  install_info = {
+    url = "https://github.com/kcl-lang/tree-sitter-kcl",
+    files = { "src/parser.c", "src/scanner.c" },
+    branch = "main",
```

Comment:
Not needed since it's the default branch.

You do need to add a lockfile entry, though.